code/modules/projectiles/gun.dm

```diff
@@ -343,7 +341,10 @@
 	var/randomized_gun_spread = 0
 	var/rand_spr = rand()
 	if(spread)
-		randomized_gun_spread =	rand(0,spread)
+		randomized_gun_spread = rand(0,spread)
+	if(HAS_TRAIT(user, TRAIT_POOR_AIM)) //nice shootin' tex
+		user.blind_eyes(1)
```

Comment:
```suggestion
		user.flash_act(override_blindness_check = TRUE, visual = TRUE, length = 5)
```
you can adjust the length as needed, but this'd implement the momentary screen flash that mothblocks asked for, I believe.

Reply:
I still prefer the increased spread PR, mind, but if someone is dead-set on merging the blindness version, I might as well improve it.

Reply:
This wouldn't be a revert then.